ext/msgpack/buffer_class.c

```diff
@@ -56,7 +56,7 @@ static size_t Buffer_memsize(const void *data)
     return sizeof(msgpack_buffer_t) + msgpack_buffer_memsize(data);
 }
 
-const rb_data_type_t buffer_data_type = {
+static const rb_data_type_t buffer_data_type = {
```

Comment:
I wonder, is the static needed given the -fvisibility=hidden below?

Reply:
Yes, but `-fvisibility=hidden` may not work on some platforms, or so I've heard. I still need to cleanup extconf to use `append_cflags` instead of `$CFLAGS <<`